scripts/scil_synthesize_b0.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Wrapper for SyNb0 available in Dipy, to run it on a single subject.
+Requires Skull-Strip b0 and t1w images as input, the script will normalize the
+t1w's WM to 110, co-register both images, then register it to the appropriate
+template, run SyNb0 and then transform the result back to the original space.
+
+This script must be used carefully, as it is not meant to be used in an
+environment with the following dependencies already installed (not default
+in Scilpy):
+- tensorflow-addons
+- tensorrt
+- tensorflow
+
+NOTE: There is no tests for this scripts due to the dependencies.
+"""
+
+
+import argparse
+import logging
+import os
+import sys
+import warnings
+
+# Disable tensorflow warnings
+with warnings.catch_warnings():
+    os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'
+    warnings.simplefilter("ignore")
+    from dipy.nn.synb0 import Synb0
+
+from dipy.align.imaffine import AffineMap
+from dipy.segment.tissue import TissueClassifierHMRF
+import nibabel as nib
+import numpy as np
+from scipy.ndimage import gaussian_filter
+
+from scilpy.io.fetcher import get_synb0_template_path
+from scilpy.io.utils import (add_overwrite_arg,
+                             add_verbose_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist)
+from scilpy.utils.image import register_image
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.RawTextHelpFormatter)
+    p.add_argument('in_b0',
+                   help='Input b0 image.')
```

Comment:
Skull-Stripped or not? You mention above that the b0 must skull-strip a few times... Please clarify... 

Idem for the T1 below